spec/models/notification_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
Can we nuke this file please? 💥 

We can always add it back in when we need to write some tests for this model.